### PR TITLE
refactor equality assertions to helper class

### PR DESCRIPTION
**Pull request by adamkorynta**
taking @MikeNeilson 's suggestion from earlier PR

### CODE REVIEW

File: cwms-data-api/src/test/java/helpers/DTOMatch.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package helpers;
```
Comment (RyanM-RMA) [inline cwms-data-api/src/test/java/helpers/DTOMatch.java]: should this be under cwms.cda.data?  Seems weird to be in the top level package to me.

**Review by RyanM-RMA — CHANGES_REQUESTED** [review]
Just a minor thing I need to know about, but it's important

File: cwms-data-api/src/test/java/helpers/DTOMatch.java (hunk 1 above)
Comment (adamkorynta) [reply cwms-data-api/src/test/java/helpers/DTOMatch.java]: I didn't see the other helper's package. Makes sense to me - moved

**Review by RyanM-RMA — DISMISSED** [review]

**Review by rma-rripken — DISMISSED** [review]

**Comment by adamkorynta** [comment]
Rebased to fix build issues

**Review by RyanM-RMA — APPROVED** [review]

**Pull request merged**